test/mocha/10-test.js

```diff
@@ -342,7 +342,8 @@ describe('Continuity Storage', () => {
         const r = await getConsensusProofPeers({blockHeight: 1, explain: true});
         const {indexName} = r.stages[0].$cursor.queryPlanner
           .winningPlan.inputStage;
-        indexName.should.equal('block.blockHeight.core.1');
+        ['block.continuity2017.blockHeight.1', 'block.blockHeight.core.1']
```

Comment:
@dlongley @mattcollier -- This is making me very nervous... the query planner is non-deterministically (it seems) switching which index it's using to answer the query.

Reply:
@msporny, yeah, this is a bit annoying. The way the query planner works is that it tries running different plans internally and then picks a winner (and constantly reevaluates from time to time) and aborts the others. Apparently, this test is running so quickly on your machine that the difference between the two indexes is in the noise when there isn't a lot of data in the database. I would think the only way to make this test consistent would be to have a sufficient number of records to ensure the proper index is used -- on all machines where these tests run.

The good news is that `getConsensusProofPeers` isn't even a thing anymore and it's on the chopping block for removal. So, I'd say just skip this test with a comment that says `getConsensusProofPeers` is being removed rather than modify the test.

Reply:
Test removed in 87135606f134433f827d4972892ebf53bf1dbd4a.